extract_assets.py

```diff
@@ -16,13 +17,13 @@ def ExtractFile(xmlPath, outputPath, outputSourcePath):
         # Don't extract if another file wasn't extracted properly.
         return
 
-    execStr = "tools/ZAPD/ZAPD.out e -eh -i %s -b baserom/ -o %s -osf %s -gsf 1 -rconf tools/ZAPDConfigs/MqDbg/Config.xml" % (xmlPath, outputPath, outputSourcePath)
-    
+    execStr = f"tools/ZAPD/ZAPD.out e -eh -i {xmlPath} -b baserom/ -o {outputPath} -osf {outputSourcePath} -gsf 1 -rconf tools/ZAPDConfigs/MqDbg/Config.xml {ZAPDArgs}"
```

Comment:
The unaccounted static flag should be used now since this PR brings that in.

Reply:
Did we end up on an agreement on that?

Reply:
There is the `--unaccounted` option already, should we make it the default?

Reply:
nothing official but I personally dont see any reason to not have them be static. if someone really wants to look for them in the map file they can just turn the flag off.

Reply:
In that case, I think `extract_assets.py` should have a flag to toggle that behaviour instead of hardcodding it on by default

Reply:
Or, extract_assets.py in the makefile can pass the ZAPD flag directly since that's what the new `-Z` flag does.